habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -507,13 +542,15 @@ def find_index_quant(minv, maxv, num_bins, value, interp=False):
             value_norm_t = index - lower
             if lower < 0:
                 min_poss_val = 0.0
-                lower = (min_poss_val - minv) * (num_bins - 1) / (maxv - minv)
+                lower = int(
+                    (min_poss_val - minv) * (num_bins - 1) / (maxv - minv)
+                )
                 value_norm_t = (index - lower) / -lower
                 lower = -1
 
             return lower, upper, value_norm_t
 
-        def comp_inter(x_i, y_i, z_i):
+        def comp_inter(x_i: int, y_i: int, z_i: int) -> int:
```

Comment:
I would rename this function to `_compute_pose_index(...)`. To be honest, I'm not sure what this really do though.